src/dstack/_internal/server/services/projects.py

```diff
@@ -180,10 +180,10 @@ async def list_user_owned_project_models(
 ) -> List[ProjectModel]:
     filters = [
         ProjectModel.owner_id == user.id,
-        ProjectModel.deleted == False,
+        ProjectModel.deleted == False,  # noqa: E712
```

Comment:
I suggest we globally ignore E712 since we do a lot of sqlalchemy comparisons where it makes sense.

Reply:
Set the E712 rule globally and deleted all `noqa` comments